File: BasicInputOutput/C++/SimpleInterest.cpp
#include <iostream>
using namespace std;

int main() {
    double Principle, rate, time;
    cout << "Enter Principle, rate, and time: ";
    cin >> Principle >> rate >> time;

    double interest = (Principle * rate * time) / 100;
    double totalAmount = interest + Principle;

    cout << "Simple Interest: " << interest << endl;
    cout << "Total Amount: " << totalAmount << endl;

    return 0;
}

File: BasicInputOutput/Java/SimpleInterest.java

import java.util.Scanner;

class SimpleInterest{
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        System.out.println("Enter Principle rate and time: ");
        double Principle = sc.nextDouble();
        double rate = sc.nextDouble();
        double time = sc.nextDouble();

        System.out.println("Simple Interest : "+(Principle*rate*time/100));
        System.out.println("Total Amount : "+(Principle*rate*time/100)+Principle);
    }
}